roles/edpm_podman/tasks/login.yml

```diff
@@ -18,7 +18,7 @@
 - name: Perform container registry login(s) with podman
   become: true
   ansible.builtin.shell: >
-    podman login --authfile ~/.config/containers/auth.json --username=$REGISTRY_USERNAME --password=$REGISTRY_PASSWORD $REGISTRY
+    podman login --authfile /etc/containers/auth.json --username=$REGISTRY_USERNAME --password=$REGISTRY_PASSWORD $REGISTRY
```

Comment:
you need to make the same change to `roles/edpm_podman/tasks/login-kube-registry.yml`

and should we make the path a variable?

Reply:
login-kube-registry now has it as well. I intend to switch to variable (with default) once I get clean molecule run.

Reply:
Variable has been introduced.